### ISSUE TITLE
Doesn't work with imported `^query`

### ISSUE THREAD

**Issue by imbolc** [Xuyuanp/sqlx-rs.nvim#1]
Thank you very much for taking the effort, I really appreciate it. Just noticed that, while while working perfectly with `sqlx::query!`, it doesn't with just `query!`. Not sure if it's intentional.

**Comment by Xuyuanp** [comment]
This is a known issue that is hard to fix. Because parsing imported queries is very complicated. Imagine that the following scenarios need to be considered.

```rust
use sqlx::query;
use sqlx::query as my_query;
use sqlx::{query, query_as};
use sqlx::{query as my_query, query_as};
```

I'm not an expert on `rust` and `treesitter`, don't known how to deal with it yet.

**Comment by imbolc** [comment]
Yeah, I don't know Treesitter either. Actually, I thought I send you a PR with using ChatGPT, but I gave up after 1.5 hours :)

Regarding `use as`, `sqlx` prefix doesn't prevent the issue completely, as you can do `use sqlx as my_sqlx` too.

Is it a big deal if a wrong string gets highlighted as sql?
